src/LibVLCSharp.Avalonia.Unofficial/VideoView.cs

```diff
@@ -0,0 +1,317 @@
+using System;
```

Comment:
So the existing videoview should be modified, to support these scenarios. Much like the WPF videoview.